riakcs/assets/configuration/spec.yaml

```diff
@@ -1,4 +1,4 @@
-name: Riak CS
+name: RiakCS
```

Comment:
FWIW, the Riak project uses "Riak CS" and we generally use the same spacing/spelling/punctuation as the product/project itself. https://docs.riak.com/riak/cs/2.1.1/index.html